.woodpecker/test.yaml

```diff
@@ -74,7 +74,7 @@ steps:
   lint-license-header:
     image: *golang_image
     commands:
-      - go install github.com/google/addlicense@latest
+      - go install github.com/google/addlicense@latest # cspell:words addlicense
```

Comment:
Same here, exlude all go install commands by a regex

Reply:
yes but the next lines not